File: 20080430/jp.go.aist.rtm.rtcbuilder/resource/CXXWin/AIST1/MyServiceSVC_impl.cpp
// -*-C++-*-
/*!
 * @file  MyServiceSVC_impl.cpp
 * @brief Service implementation code of MyService.idl
 *
 */

#include "MyServiceSVC_impl.h"

/*
 * Example implementational code for IDL interface MyService
 */
MyServiceSVC_impl::MyServiceSVC_impl()
{
  // Please add extra constructor code here.
}


MyServiceSVC_impl::~MyServiceSVC_impl()
{
  // Please add extra destructor code here.
}


/*
 * Methods corresponding to IDL attributes and operations
 */
char* MyServiceSVC_impl::echo(const char* msg)
{
  // Please insert your code here and remove the following warning pragma
  #pragma message("Code missing in function <char* MyServiceSVC_impl::echo(const char* msg)>")
  return NULL;
}

EchoList* MyServiceSVC_impl::get_echo_history()
{
  // Please insert your code here and remove the following warning pragma
  #pragma message("Code missing in function <EchoList* MyServiceSVC_impl::get_echo_history()>")
  return NULL;
}

void MyServiceSVC_impl::set_value(CORBA::Float value)
{
  // Please insert your code here and remove the following warning pragma
  #pragma message("Code missing in function <void MyServiceSVC_impl::set_value(CORBA::Float value)>")
}

CORBA::Float MyServiceSVC_impl::get_value()
{
  // Please insert your code here and remove the following warning pragma
  #pragma message("Code missing in function <CORBA::Float MyServiceSVC_impl::get_value()>")
  return NULL;
}

ValueList* MyServiceSVC_impl::get_value_history()
{
  // Please insert your code here and remove the following warning pragma
  #pragma message("Code missing in function <ValueList* MyServiceSVC_impl::get_value_history()>")
  return NULL;
}



// End of example implementational code

File: 20080430/jp.go.aist.rtm.rtcbuilder/resource/Java/ML/ML3/MyServiceSVC_impl.java
// -*-Java-*-
/*!
 * @file  MyServiceSVC_impl.java
 * @brief Service implementation code of MyServiceType1.idl
 *
 */
/*!
 * @class MyServiceSVC_impl
 * Example class implementing IDL interface MyService
 */
public class MyServiceSVC_impl extends MyServicePOA{
    
    public MyServiceSVC_impl() {
        // Please add extra constructor code here.
    }

    /*
     * Methods corresponding to IDL attributes and operations
     */
    public String echo(TestStruct msg) {
        // Please insert your code here and remove the following warning pragma
        // TODO "Code missing in function <String echo(TestStruct msg)>"
        return null;
    }

    public String[] get_echo_history(String[] echoList) {
        // Please insert your code here and remove the following warning pragma
        // TODO "Code missing in function <EchoList get_echo_history(EchoList echoList)>"
        return null;
    }

    public void set_value(float value) {
        // Please insert your code here and remove the following warning pragma
        // TODO "Code missing in function <void set_value(float value)>"
    }

    public float get_value(TestStructHolder value) {
        // Please insert your code here and remove the following warning pragma
        // TODO "Code missing in function <float get_value(TestStruct value)>"
        return 0;
    }

    public float[] get_value_history() {
        // Please insert your code here and remove the following warning pragma
        // TODO "Code missing in function <ValueList get_value_history()>"
        return null;
    }

    public float get_test_value(float test) {
        // Please insert your code here and remove the following warning pragma
        // TODO "Code missing in function <Values get_test_value(Values test)>"
        return 0;
    }

    public void get_test_value2(TestStructHolder test) {
        // Please insert your code here and remove the following warning pragma
        // TODO "Code missing in function <void get_test_value2(StrVal test)>"
    }

//  End of example implementational code
}
